js/views/userPageVw.js

```diff
@@ -307,6 +307,25 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (self.model.get('page').profile.handle) {
+            var profile = self.model.get('page').profile,
```

Comment:
I'm not too worried about running out of localStorage space, but we also don't want to invite trouble with potentially infinite lists of things. Capping it at 1000 would be fine.